e2e_test/sink/clickhouse_sink.slt

```diff
@@ -17,6 +17,7 @@ CREATE SINK s6 AS select mv6.v1 as v1, mv6.v2 as v2, mv6.v3 as v3, mv6.v4 as v4,
     clickhouse.password = '',
     clickhouse.database = 'default',
     clickhouse.table='demo_test',
+    commit_checkpoint_interval = 1,
```

Comment:
Is setting this field just to implicitly enable sink decouple, or it becomes required to set this field after this PR?

Reply:
Because sink_decouple = false will report an error when commit_checkpoint_interval >10;
And in ci , we need  sink_decouple = false to ensure that flush can flush data to sink.
 So we set commit_checkpoint_interval = 1, 
 After this PR, if it is not set, a default value of 10 will be given